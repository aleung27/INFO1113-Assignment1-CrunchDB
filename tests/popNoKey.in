POP l
SET L 9
POP l
POP L
GET L
POP L
LIST ENTRIES
