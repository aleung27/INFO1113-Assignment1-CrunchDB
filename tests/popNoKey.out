> no such key

> ok

> no such key

> 9

> []

> nil

> L []

